cmd/clusterawsadm/cloudformation/bootstrap/template.go

```diff
@@ -194,6 +195,14 @@ func (t Template) RenderCloudFormation() *cloudformation.Template {
 		}
 	}
 
+	if t.Spec.EKS.EnableNodesAndWorkloadsPolicy {
+		template.Resources[string(EKSViewNodesAndWorkloadsPolicy)] = &cfn_iam.ManagedPolicy{
+			ManagedPolicyName: t.NewManagedName("eks-view-nodes-workloads"),
```

Comment:
I think we should shorten the name of this as its going to have a suffix appended and optionally a prefix which will make it long.

Perhaps something like `eks-console`?